Packs/AlienVault_USM_Anywhere/Integrations/AlienVault_USM_Anywhere/AlienVault_USM_Anywhere.py

```diff
@@ -136,6 +137,24 @@ def get_time_range(time_frame=None, start_time=None, end_time=None):
     return date_to_timestamp(start_time), date_to_timestamp(end_time)
 
 
+def convert_timestamp_to_iso86(timestamp: str, timezone_letter: str = 'Z') -> str:
+    """ Convert timestamp from AlienVault to iso86 format
+
+    Args:
+        timestamp: The timestamp as received from AlienVault
+        timezone_letter: The timezone letter as received from AlienVault
+
+    Returns: The timestamp in iso86 format, similar to what AlienVault returns itself.
+
+    """
+    if not timestamp:
+        return ''
+    datetime_from_timestamp = dateparser.parse(timestamp, settings={"TO_TIMEZONE": timezone_letter,
+                                                                    "RETURN_AS_TIMEZONE_AWARE": True})
+    time_in_iso86 = datetime_from_timestamp.strftime("%Y-%m-%dT%H:%M:%S.%f")
+    return time_in_iso86[:-3] + timezone_letter
```

Comment:
i think you can just use the `%z` or `%Z` in `strftime` instead of manipulating the string